cypress/e2e/data-import/settings/mrc-import-for-update-instance-with-protected-fields-after-editing-marc-bib-in-quickMarc.cy.js

```diff
@@ -0,0 +1,223 @@
+import getRandomPostfix from '../../../support/utils/stringTools';
+import permissions from '../../../support/dictionary/permissions';
+import TestTypes from '../../../support/dictionary/testTypes';
+import DevTeams from '../../../support/dictionary/devTeams';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import Z3950TargetProfiles from '../../../support/fragments/settings/inventory/z39.50TargetProfiles';
+import MarcFieldProtection from '../../../support/fragments/settings/dataImport/marcFieldProtection';
+import MatchProfiles from '../../../support/fragments/data_import/match_profiles/matchProfiles';
+import NewMatchProfile from '../../../support/fragments/data_import/match_profiles/newMatchProfile';
+import FieldMappingProfiles from '../../../support/fragments/data_import/mapping_profiles/fieldMappingProfiles';
+import NewFieldMappingProfile from '../../../support/fragments/data_import/mapping_profiles/newFieldMappingProfile';
+import NewActionProfile from '../../../support/fragments/data_import/action_profiles/newActionProfile';
+import ActionProfiles from '../../../support/fragments/data_import/action_profiles/actionProfiles';
+import NewJobProfile from '../../../support/fragments/data_import/job_profiles/newJobProfile';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import TopMenu from '../../../support/fragments/topMenu';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import InventoryEditMarcRecord from '../../../support/fragments/inventory/inventoryEditMarcRecord';
+import InventoryViewSource from '../../../support/fragments/inventory/inventoryViewSource';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import FileDetails from '../../../support/fragments/data_import/logs/fileDetails';
+import FileManager from '../../../support/utils/fileManager';
+import Users from '../../../support/fragments/users/users';
+
+describe('ui-data-import', () => {
+  let user = null;
+  let instanceHrid;
+  const marcFieldProtectionId = [];
+  const authentication = '100473910/PAOLF';
+  const protectedFields = {
+    firstField: '*',
+    secondField: '920'
+  };
+  const quantityOfItems = '1';
+  const oclcForImport = '830936944';
+  // unique file names
+  const editedMarcFileName = `C356829 editedMarcFile.${getRandomPostfix()}.mrc`;
+  const nameMarcFileForUpload = `C356829 autotestFile.${getRandomPostfix()}.mrc`;
+  // unique profile names
+  const matchProfileName = `C356829 001 to Instance HRID ${getRandomPostfix()}`;
```

Comment:
Please add this string directly to the const matchProfile = {
    profileName:. Why have you created a variable?

Reply:
I need the variable because I use it in MatchProfiles.deleteMatchProfile(matchProfileName); in the after block